duneana/CAFMaker/CAFMaker_module.cc

```diff
@@ -75,11 +85,20 @@ namespace caf {
 
 
     private:
-      void AddGlobalTreeToFile(TFile* f, SRGlobal& global);
-      void FillTruthInfo(caf::StandardRecord& sr,
-                         std::vector<simb::MCTruth> const& truth,
+      void FillTruthInfo(caf::SRTruthBranch& sr,
+                         std::vector<simb::MCTruth> const& mctruth,
+                         std::vector<simb::GTruth> const& gtruth,
                          std::vector<simb::MCFlux> const& flux,
-                         art::Event const& evt) const;
+                         art::Event const& evt);
+
+      void FillMetaInfo(caf::SRDetectorMeta &meta, art::Event const& evt) const;
+      void FillBeamInfo(caf::SRBeamBranch &beam, const art::Event &evt) const;
+      void FillRecoInfo(caf::SRCommonRecoBranch &recoBranch, const art::Event &evt) const;
+      void FillCVNInfo(caf::SRCVNScoreBranch &cvnBranch, const art::Event &evt) const;
+      void FillEnergyInfo(caf::SRNeutrinoEnergyBranch &ErecBranch, const art::Event &evt) const;
+      void FillRecoParticlesInfo(caf::SRRecoParticlesBranch &recoParticlesBranch, const art::Event &evt) const;
+      void FillDirectionInfo(caf::SRDirectionBranch &dirBranch, const art::Event &evt) const;
+      int FillGENIERecord(simb::MCTruth const& mctruth, simb::GTruth const& gtruth);
 
       std::string fMVASelectLabel;
       std::string fMVASelectNueLabel;
```

Comment:
The MVA PID isn't actually used in any of the code, or stored in the SR, so maybe remove all the MVA bits?